backend/src/main/java/pl/edu/pw/ee/backend/api/order/BazaZamowien.java

```diff
@@ -43,60 +39,32 @@ public int setOrderData(OrderDTO orderData) {
 
         log.info("Retrieving client with id: {}", orderDataDto.clientId());
 
-        Client client = clientRepository.findById(orderDataDto.clientId())
-                .orElseThrow(() -> new ClientNotFoundException(orderDataDto.clientId()));
+        Client client = clientService.findClientById(orderDataDto.clientId());
 
         OrderData orderDataToSave = buildOrderDataToSave(orderDataDto, client);
 
         log.info("Retrieving {} dishes", meals.size());
 
         List<Dish> dishes = meals.stream()
-                .map(meal -> dishRepository.findById(meal.dish().dishId())
-                        .orElseThrow(() -> new DishNotFoundException(meal.dish().dishId())))
+                .map(meal -> dishService.findDishById(meal.dish().dishId()))
                 .toList();
 
         Order orderToSave = buildOrderToSave(orderDataToSave, dishes);
```

Comment:
To tak samo nwm czy to trzeba zmergowac czy nie.

Reply:
Jeżeli uznamy, że te serwisy to takie wrappery na repozytoria to ich odpowiedzialnością by było to, że przekonwertują dane w taki sposób żeby można było je zapisać w bazie danych ale tbh nwm.